src/config/offlineDB.connection.js

```diff
@@ -0,0 +1,8 @@
+import Dexie from 'dexie'
+
+const dbInstance = new Dexie(process.env.VUE_APP_OFFLINE_DB || 'tknbrgoff')
```

Comment:
I'd maybe add another `const DEFAULT_DB_NAME = 'tknbrgoff'` on top of this declaration (so it's clear what it means), or something like that (not sure if it refers to its name)